docs/open-source-project/release-notes/tizen-4-0-m3.md

```diff
@@ -0,0 +1,258 @@
+# Tizen 4.0 Public M3 Release Notes
+
+Release Date: Aug. 31, 2018
+
+
+## System (Kernel and System Framework)
+
+### Fixes
+
+- Security improvement
+  - Systemd patch has been applied for CVE-2018-1049.
+
+
+## System (Base)
+
+### New and changed features
+
+- The following Open-source libraries have been upgraded:
+  - file (5.32)
+    - CVE-2018-10360
+  - sqlite (3.21.0)
+    - CVE-2018-8740
+  - libxml (2.9.7)
+    - CVE-2018-9251
+  - libxslt (1.1.31)
+  - procps-ng (3.3.15)
+    - CVE-2018-1126, CVE-2018-1125, CVE-2018-1124, CVE-2018-1123, CVE-2018-1122
+  - tzdata (2017c)
+  - unzip (6.10c23)
+    - CVE-2018-1000031, CVE-2018-1000032, CVE-2018-1000033
+  - vim (8.0.586)
+
+
+## Application Framework
+
+### New and changed features
+
+- TIDL – Tizen Interface Definition Language has been added (Mobile profile)
+  - TIDL supports app-to-app RPC style IPC
+  - rpc-port API has been added to support TIDL
+
+
+## Window System
+
+### New and changed features
+
+- Extended Wayland Protocols
+  - The ‘event_surface’ event has been added to tizen_keyrouter protocol for sending the surface information before sending the a key to client(s).
+- Enlightenment Wayland Display Server
+  - A D-Bus policy has been updated to enhance security regarding D-Bus interface.
+  - A new layer has been added for e_desk_object.
+  - New hooks regarding HW compositing have been provided for vendor's enlightenment module(s) to handle HWC policy properly.
+- Tizen buffer HAL
+  - Add the tbm_dummy_display.
+  - Add the tbm_surface_internal_set/get_damage.
+
+### Fixes
+
+- Enlightenment Wayland Display Server
+  - The bugs regarding sending pre-unobscured visibility event have been fixed.
+  - The bugs related to uniconify_render have been fixed.
+  - The bug not returning to the default screen mode has been fixed.
+  - The error that the quick panel doesn’t disappear has been fixed.
```

Comment:
The error that the quick panel not disappearing  has been fixed.

Reply:
I'm just wondering your recommendation is grammatically correct.
Shouldn't it be used as below?
1) The error that the quick panel is not disappearing has been fixed.
or
2) The error that the quick panel doesn’t disappear has been fixed.

Anyway if you reconfirm that your recommendation is correct, we will accept your recommendation below.
The error that the quick panel not disappearing has been fixed.